tests/test_regex.py

```diff
@@ -50,6 +50,30 @@ def test_dom_mutation_fail():
     assert _do_test_raw('foo.DOMAttrModified = bar;').failed()
 
 
+def test_bug_1200929():
+    """Test that XPIProvider and AddonManagerInternal are not used."""
+
+    real_world_usage = (
+        'XPIProviderBP = Cu.import("resource://gre/modules/XPIProvider.jsm", {});')
+
+    err = _do_test_raw(real_world_usage)
+    assert err.failed()
+    assert err.signing_summary['high'] == 1
+
+    err = _do_test_raw('// XPIProvider')
```

Comment:
So this is supposed to fail because it's a regex test, and it can't detect if it's in a comment, right? If so, I'd simply drop this test, because it's not something we _want_, it's something we have, and we don't care about.